src/app/ApplicationTemplate.Presentation/Configuration/ApiConfiguration.cs

```diff
@@ -55,18 +53,6 @@ private static IServiceCollection AddUserProfile(this IServiceCollection service
 		return services.AddSingleton<IUserProfileRepository, UserProfileRepositoryMock>();
 	}
 
-	private static IServiceCollection AddMinimumVersion(this IServiceCollection services)
```

Comment:
Don't you need to keep those for the functional tests to work?

Reply:
Functional tests don't use anything from the View layer. So I guess this adds to my original question. 😋